2023_VillaIslas_Science/2023_VillaIslas_Science.janno

```diff
@@ -0,0 +1,28 @@
+Poseidon_ID	Genetic_Sex	Group_Name	Country	Country_ISO	Location	Site	Latitude	Longitude	Date_Type	Date_BC_AD_Start	Date_BC_AD_Median	Date_BC_AD_Stop	Date_Note	MT_Haplogroup	Y_Haplogroup	Source_Tissue	Nr_Libraries	Library_Names	Capture_Type	UDG	Library_Built	Genotype_Ploidy	Data_Preparation_Pipeline_URL	Endogenous	Nr_SNPs	Damage	Contamination	Contamination_Err	Contamination_Meas	Contamination_Note	Genetic_Source_Accession_IDs	Publication	Eager_ID	Main_ID	RateErrX	RateErrY	RateX	RateY
```

Comment:
I think the `Library_Names` column is filled incorrectly. It should include "identifiers of the libraries used to generate the genotype data for the sample", but here it has entries like "mtDNA-genome;Whole-genome". These are not library IDs. What does that mean here?

Reply:
`Library_Names` is actually filled automatically from the ENA metadata.
I would be against updating these, as it muddles the transparency of the proccessing.